Project.toml

```
@@ -17,14 +17,13 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
 AbstractTrees = "0.3"
-DataFrames = "0.19.4, 0.20"
-DataStructures = "0.17.3, 0.17"
+DataFrames = "0.19.4, 0.20, 1.2"
+DataStructures = "0.17.3, 0.17, 0.18"
 ImageFiltering = "0.6.5, 0.6"
 LeftChildRightSiblingTrees = "0.1"
-OffsetArrays = "0.11.3, 1"
+OffsetArrays = "0.11.3, 1, 1.10"
```

and also

```suggestion
OffsetArrays = "0.11.3, 1"
```

since `"1"` is equivalent to `"1, 1.10"`